app/views/main.scala.html

```diff
@@ -49,5 +49,7 @@
 
         @fragments.global.footer(product, edition)
         @fragments.footerJavaScript()
+        <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
+         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
```

Comment:
You could add the script to the DOM inside the init of awin.js, then if the user has opted out via DNT, or we're in test mode and so analyticsEnabled() returns false,  we won't load the JS from dwin1.com